File: SDKs/amazonaws.com/lightsail/2016-11-28/python/src/sdk/models/shared/containerserviceslistresult.py
from dataclasses import dataclass, field
from typing import List,Optional
from dataclasses_json import dataclass_json
from sdk import utils
from . import *


@dataclass_json
@dataclass
class ContainerServicesListResult:
    container_services: Optional[List[ContainerService]] = field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('containerServices') }})
    

File: SDKs/amazonaws.com/lightsail/2016-11-28/typescript/src/sdk/models/shared/containerserviceslistresult.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";
import { ContainerService } from "./containerservice";



export class ContainerServicesListResult extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=containerServices", elemType: ContainerService })
  containerServices?: ContainerService[];
}
